news/2019-02-24-project-loved-week-of-february-24th.md

```diff
@@ -0,0 +1,206 @@
+
+## <a name="osumania" id="osumania"></a>osu!mania
+
+osu!mania Loved candidates were chosen by [Lenfried-](https://osu.ppy.sh/users/5314573), [Pope Gadget](https://osu.ppy.sh/users/2288341), [Shoegazer](https://osu.ppy.sh/users/2520707) and [Toaph Daddy](https://osu.ppy.sh/users/7616811)!
+
+---
+
+[![](/wiki/shared/news/2019-02-24-project-loved-week-of-february-24th/mania/cyber-induction.jpg)](https://osu.ppy.sh/community/forums/topics/873720)
+
+[Sharpnel Sounds - Cyber Induction](https://osu.ppy.sh/beatmapsets/827679#mania) by [IcyWorld](https://osu.ppy.sh/users/1383951)
+
+Is there anything else that needs to be said about Cyber Induction? If osu!mania had a city, Cyber Induction would be, with no exaggeration, \*the\* iconic building downtown.
+
+A chart that was made all the way back in 2011, Cyber Induction has gained popularity and even notoriety for being a staple chart for competitive gameplay since 2015. The chart is extremely straightforward; it is jumpstream-focused with no major difficulty spikes and miscellaneous strings attached. The chart's simplicity makes it an excellent gauge of jumpstream and JS stamina ability, distinctly better than most JS charts out there in that regard.
+
+This mapset also comes with four other difficulties: four sped-up versions of the original Cyber Induction chart (1.1x to 1.4x rates), making it not only a more competitive mapset, but it makes the mapset far more accessible than what it already is with a single Cyber Induction chart.
+
+[![](/wiki/shared/news/2019-02-24-project-loved-week-of-february-24th/mania/rockhill.jpg)](https://osu.ppy.sh/community/forums/topics/873719)
+
+[Beltaine - Rockhill](https://osu.ppy.sh/beatmapsets/464358#mania) by [Ryu Sei](https://osu.ppy.sh/users/3427638)
+
+Simplicity is the key to success for Rockhill, the next of our mania nominations. The staple of this chart comes from the use of single LNs to follow the violin, with some additional layering being compounded as the song continues, introducing heavier difficulty and tricky releases. This makes for a fun and dynamic experience, while not taking away the charm of it's initial simplicity. These facts have it a very sizable following, stacking with it's all approachable nature and allowing for an extreme amount of support, only aided by its use in the 2017 osu!mania world cup. All of this more than proves its worth for the Loved section.
```

Comment:
Suggested diff does not show.
`2017 osu!mania world cup` -> `osu!mania World Cup 2017`
```suggestion
Simplicity is the key to success for Rockhill, the next of our mania nominations. The staple of this chart comes from the use of single LNs to follow the violin, with some additional layering being compounded as the song continues, introducing heavier difficulty and tricky releases. This makes for a fun and dynamic experience, while not taking away the charm of it's initial simplicity. These facts have it a very sizable following, stacking with it's all approachable nature and allowing for an extreme amount of support, only aided by its use in the osu!mania World Cup 2017. All of this more than proves its worth for the Loved section.
```